xml/ha_qdevice-qnetd.xml

```diff
@@ -0,0 +1,508 @@
+               </para>
+            </callout>
+            <callout arearefs="co-quorum-nodesnumber">
+               <para>
+                  The number of nodes we are expecting. In this example, it is a
+                  two-node cluster.
+               </para>
+            </callout>
+            <callout arearefs="co-quorum-nodeid">
+               <para>
+                  In this case, since it is not explicitly specified in
+                  <filename>corosync.conf</filename>, this ID is an 32-bit integer
+                  representation of the IP address. In this example, the value
+                  <literal>3232235777</literal> stands for the IP address <systemitem
+                     class="ipaddress">&subnetI;.1</systemitem>.
+               </para>
+            </callout>
+            <callout arearefs="co-quorum-quorate">
+               <para>
+                  The quorum status. In this case, the cluster has quorum.
+               </para>
+            </callout>
+            <callout arearefs="co-quorum-nodes">
+               <para>
+                  The status for of each cluster nodes means:
+               </para>
+               <variablelist>
+                  <varlistentry>
+                     <term><literal>A</literal> (Alive) or <literal>NA</literal> (not alive)</term>
+                     <listitem>
+                        <para>Shows the status between &qdevice; and &corosync;.
+                        </para>
+                     </listitem>
+                  </varlistentry>
+                  <varlistentry>
+                     <term><literal>V</literal> (Vote) or <literal>NV</literal> (non vote)</term>
+                     <listitem>
+                        <para>
+                           Shows if the quorum device has given vote (letter <literal>V</literal>)
+                           to the node.
+                           A letter <literal>V</literal> means that both nodes can communicate
+                           with each other. In a split-brain situation, one node would be
+                           set to <literal>V</literal> and the other node would be set to
+                           <literal>NV</literal>.
+                        </para>
+                     </listitem>
+                  </varlistentry>
+                  <varlistentry>
+                     <term><literal>MW</literal> (Master wins) or
+                           <literal>NMW</literal>(not master wins)</term>
+                     <listitem>
+                        <para>
+                           Shows if the quorum device <parameter>master_wins</parameter>
+                           flag is set. By default you see <literal>NMW</literal> (not master
+                           wins) as the flag is not set.
+                           See the man page votequorum_qdevice_master_wins(3) for more
+                           information.
```

Comment:
```suggestion
                           flag is set. By default, the flag is not set, so you see <literal>NMW</literal> (not master
                           wins)
                           See the man page votequorum_qdevice_master_wins(3) for more
                           information.
```

Reply:
For markup/references to man pages, also see the style guide: https://documentation.suse.com/style/current/single-html/docu_styleguide/index.html#sec-other-external